src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+class FigureSupplier {
```

Comment:
Classes should not begin with an empty line. Remove the empty line to adhere to the coding standards.